app/models/create_models.py

```diff
@@ -0,0 +1,17 @@
+from app.models.models import Knight
+
+
+def create_knight(knight: dict) -> Knight:
+    result = Knight(
```

Comment:
`result` is not a descriptive variable name for `Knight` instance, change it to another one.